source/less/blocks/advantages.less

```
@@ -22,42 +38,111 @@
 
 .advantages__list {
   .list-reset;
+
+  @media (min-width: @tablet-width) {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    max-width: 1200px;
+    margin: 0 auto;
+  }
 }
 
 .advantages__item {
   padding: 25px 0 23px;
   background-color: @gray;
   text-align: center;
 
+  @media (min-width: @tablet-width) {
+    padding-top: 42px;
+
+    &:nth-of-type(1){
+      grid-column:  1/4;
```

чуть более универсальный способ
```suggestion
      grid-column:  1/-1;
```